packages/desktopjs/tests/unit/container.spec.ts

```diff
@@ -193,6 +193,17 @@ describe("container", () => {
                 container.loadLayout("Test");
             });
 
+            it ("loadLayout with layout creates window", async () => {
+                const layoutToLoad: PersistedWindowLayout =  new PersistedWindowLayout("Test");
+                layoutToLoad.windows.push({ name: "1", id: "1", url: "url", bounds: {}, state: { "value": "foo" }, group: ["1", "2", "3"]});
```

Comment:
We should have a test for trying to load an invalid state.

Reply:
Example?  Something that isn't a layout?

Reply:
Like passing a number or another type?  Also something that is duck typed but not exact?

Reply:
Or passing bounds twice, etc